src/ray/gcs/accessor.h

```diff
@@ -560,6 +560,20 @@ class NodeInfoAccessor {
   /// \return Status
   virtual Status AsyncReSubscribe() = 0;
 
+  /// Set the Intrenal Config string from the first node.
```

Comment:
```suggestion
  /// Set the internal config string that will be used by all nodes started in the cluster.
```

Reply:
It's just an artifact of how we currently start ray that this is the first node, it could be set independently in the future.